tools/env/dev.ts

```diff
@@ -5,7 +5,7 @@ const DevConfig: EnvConfig = {
   RMMAPI: 'http://testdata.nist.gov/rmm/',
   SDPAPI: 'http://testdata.nist.gov/sdp/',
   PDRAPI: 'http://testdata.nist.gov/pdr/',
-  DISTAPI: 'http://testdata.nist.gov/ds/',
+  DISTAPI: 'http://testdata.nist.gov/od/',
```

Comment:
This should probably be /od/ds/ instead and have the application adjust accordingly.